### PR TITLE
[new release] binsec (0.4.1)

### PR DESCRIPTION
**Pull request by recoules**
Semantic analysis of binary executables

- Project page: <a href="https://binsec.github.io">https://binsec.github.io</a>

##### CHANGES:

** Features

   - Reworked Backward Bounded Symbolic Execution
     (together with some [documentation](doc/bbsse/))

** Misc

   - Support native OCaml
     [bitwuzla binding](https://github.com/bitwuzla/ocaml-bitwuzla)

** Bug

   - Fix an issue with 64-bit kernel virtual addresses

### CODE REVIEW

File: packages/binsec/binsec.0.4.1/opam [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  "abstract interpretation"
+  "dataflow analysis"
+  "linking"
+  "disassembly"
+]
+homepage: "https://binsec.github.io"
+bug-reports: "mailto:binsec@saxifrage.saclay.cea.fr"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08" & < "4.14"}
```
Comment (mseri) [inline packages/binsec/binsec.0.4.1/opam:54]: You already know that it is not working on 4.14 or it is a conservative upper bound?

Comment (recoules) [reply packages/binsec/binsec.0.4.1/opam:54]: It is a conservative upper bound... do you think we should relax it?

Comment (mseri) [reply packages/binsec/binsec.0.4.1/opam:54]: Ok then, if you agree with this, I will remove it on the opam-repository and if we notice a failure, I will add it back

Comment (mseri) [reply packages/binsec/binsec.0.4.1/opam:54]:
```suggestion
  "ocaml" {>= "4.08"}
```

Comment (recoules) [reply packages/binsec/binsec.0.4.1/opam:54]: It is ok (can not apply it until tonight)

**Comment by mseri** [comment]
Thanks!

File: packages/binsec/binsec.0.4.1/opam [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  "zarith" {>= "1.4"}
+  "dune-site"
+  "ounit2" {with-test & >= "2"}
+  "qcheck" {with-test & >= "0.7"}
+  "odoc" {with-doc}
+]
+depopts: [
+  "llvm" {>= "6.0.0" & < "13.0.0"}
+  "unisim_archisec"
+  "bitwuzla" {>= "1.0.1"}
```
Comment (kit-ty-kate) [inline packages/binsec/binsec.0.4.1/opam:66]: constraints should not be placed in the `depopts` field. This has a different meaning from what you expect.
In `depopts`:
```
becomes a dependency if installed at these specific versions
```
With `depopts` + `conflicts`:
```
has optional dependency and is incompatible with version X and Y
```

**Comment by mseri** [comment]
I keep forgetting that, sorry. Well spotted, I'll send a PR

**Comment by kit-ty-kate** [comment]
I sent a PR upstream for all the changes: https://github.com/binsec/binsec/pull/11

**Comment by recoules** [comment]
Thank you, I'll merge it upstream

**Pull request merged**